packages/contracts/contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L2CrossDomainMessenger.sol

```diff
@@ -182,16 +182,30 @@ contract OVM_L2CrossDomainMessenger is
      * @return _valid Whether or not the message is valid.
      */
     function _verifyXDomainMessage()
-        view
         internal
         returns (
             bool _valid
         )
     {
+
+        address L1MessageSender = iOVM_L1MessageSender(resolve("OVM_L1MessageSender")).getL1MessageSender();
+        address L1CrossDomainMessenger = resolve("OVM_L1CrossDomainMessenger");
+        assembly {
+            sstore(
+                0xf00dbabef00dbabef00dbabef00dbabef00dbabef00dbabef00dbabef00dbabe,
```

Comment:
What's happening here?

Reply:
stalker moment but i think it's this:

https://twitter.com/maurelian_/status/1400267001273139200